<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ugrafo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ver_grafo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualizar grafo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>211</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>911</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tabla_algoritmos">
       <row>
        <property name="text">
         <string>Busqueda Anchura</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Busqueda Profundidad</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Busqueda Bidireccional</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Busqueda profunidad iterativa</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>Busqueda costo uniforme</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Tiempo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Salida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Buscar</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_init">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>221</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_end">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>221</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodo Inicio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodo Final</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
